DependencyInjection/SonataUserExtension.php

```diff
@@ -15,18 +15,29 @@
 use Symfony\Component\Config\Definition\Processor;
 use Symfony\Component\Config\FileLocator;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface;
 use Symfony\Component\DependencyInjection\Loader\XmlFileLoader;
 use Symfony\Component\DependencyInjection\Reference;
 use Symfony\Component\HttpKernel\DependencyInjection\Extension;
 
 /**
  * @author     Thomas Rabaix <thomas.rabaix@sonata-project.org>
  */
-class SonataUserExtension extends Extension
+class SonataUserExtension extends Extension implements PrependExtensionInterface
 {
     /**
      * {@inheritdoc}
      */
+    public function prepend(ContainerBuilder $container)
+    {
+        if ($container->hasExtension('twig')) {
+            $container->prependExtensionConfig('twig', ['form_themes' => ['SonataUserBundle:Form:form_admin_fields.html.twig']]);
+        }
+    }
+
+    /**
+     * {@inheritdoc}
+     */
```

Comment:
Is there no need to remove https://github.com/sonata-project/SonataUserBundle/blob/c3229613121efbf8f0f8f09e2ec79d8476c4a165/DependencyInjection/SonataUserExtension.php#L81-L85?

Reply:
Yes, sure, it has to be removed. It is done now.